ci/cvmfs/container.sh

```diff
@@ -64,7 +49,7 @@ else
 fi
 
 ${CVMFS_SOURCE_LOCATION}/packaging/container/build.sh \
-  ${CVMFS_SOURCE_LOCATION} ${CVMFS_RESULT_LOCATION} ${CVMFS_BUSYBOX} ${CVMFS_TAG} \
```

Comment:
I think we want both the `CVMFS_TAG` and the `git_hash`.  The `CVMFS_TAG` provides us a linear version number, which is nice, and the git hash allows us to clone the corresponding sources.